requirements.txt

```diff
@@ -97,3 +97,6 @@ ciso8601==1.0.5
 # Reviews requirements
 transitions==0.6.1
 enum34==1.1.6
+
+# newrelic APM agent
+newrelic==2.60.0.46
```

Comment:
Is this necessary, since we already have this in `release.txt.`, which is installed on server deployments?